usaspending_api/api_contracts/contracts/v2/download/awards.md

```diff
@@ -139,7 +139,8 @@ This route sends a request to the backend to begin generating a zipfile of award
 + `award_type_codes` (optional, array[string])
 + `agencies` (optional, array[Agency], fixed-type)
 + `contract_pricing_type_codes` (optional, array[string])
-+ `elasticsearch_keyword` (optional, string)
```

Comment:
Renamed this because the previous name was terribly not descriptive. I concluded this change was OK because this filter is only used by our code base (internally). Even if a request comes in with this filter, it won't be used.